spec/support/magi_medicaid/mitc/eligibility_request.json

```diff
@@ -1,134 +1,130 @@
 {
-  "State":"ID",
-  "Application Year":2014,
-  "Name":"application example",
-  "People":[
-     {
-        "Is Applicant":"Y",
-        "Applicant Attest Blind or Disabled":"Y",
-        "Student Indicator":"Y",
-        "Medicare Entitlement Indicator":"Y",
-        "Incarceration Status":"Y",
-        "Lives In State":"Y",
-        "Claimed as Dependent by Person Not on Application":"Y",
-        "Applicant Attest Long Term Care":"Y",
-        "Has Insurance":"Y",
-        "State Health Benefits Through Public Employee":"Y",
-        "Prior Insurance":"Y",
-        "Applicant Pregnant Indicator":"Y",
-        "Applicant Post Partum Period Indicator":"N",
-        "Former Foster Care":"Y",
-        "Required to File Taxes":"Y",
-        "US Citizen Indicator":"Y",
-        "Lawful Presence Attested":"Y",
-        "Immigration Status":"01",
-        "Refugee Status":"N",
-        "Amerasian Immigrant":"Y",
-        "Veteran Status":"Y",
-        "Victim of Trafficking":"Y",
-        "Seven Year Limit Start Date":"2001-07-01",
-        "Five Year Bar Applies":"Y",
-        "Five Year Bar Met":"Y",
-        "Applicant Has 40 Title II Work Quarters":"Y",
-        "Person ID":1,
-        "Applicant Age":27,
-        "Hours Worked Per Week":20,
-        "Claimer Is Out of State":"Y",
-        "Prior Insurance End Date":"2013-06-01",
-        "Number of Children Expected":1,
-        "Had Medicaid During Foster Care":"Y",
-        "Age Left Foster Care":22,
-        "Foster Care State":"ID",
+  "State": "ID",
+  "Application Year": 2014,
+  "Name": "application example",
+  "People": [
+    {
+      "Is Applicant": "Y",
+      "Applicant Attest Blind or Disabled": "Y",
+      "Student Indicator": "Y",
+      "Medicare Entitlement Indicator": "Y",
+      "Incarceration Status": "Y",
+      "Lives In State": "Y",
+      "Claimed as Dependent by Person Not on Application": "Y",
+      "Applicant Attest Long Term Care": "Y",
+      "Has Insurance": "Y",
+      "State Health Benefits Through Public Employee": "Y",
+      "Prior Insurance": "Y",
+      "Applicant Pregnant Indicator": "Y",
+      "Applicant Postpartum Period Indicator": "N",
```

Comment:
_MAGI In The Cloud_ expects us to send the attribute name as 'Applicant Post Partum Period Indicator'. Please revert this change.